ic-http-mock/example_canister/src/canister_backend/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "canister_backend"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+candid = "0.8.2"
+ic-cdk = "0.6.0"
+ic-cdk-macros = "0.6.0"
+ic-http-mock = {path = "../../../"}
```

Comment:
nit: It's odd to me that a mocking library is included as dependency of the canister. I would expect mocking libraries only to be included as dev-dependencies.

Reply:
Done, renamed `ic-http-mock` to `ic-http` which provides a simple http outcall API with and extra convenience of mocking it in tests.